compiler/Cargo.toml

```diff
@@ -11,14 +11,20 @@ documentation = "https://docs.rs/grpcio-compiler"
 description = "gRPC compiler for grpcio"
 categories = ["network-programming"]
 
+[features]
+default = ["protobuf-codec"]
+protobuf-codec = ["protobuf-codegen"]
+prost-codec = ["prost-build", "prost-types", "prost"]
+
 [dependencies]
-protobuf = "2.0"
-protobuf-codegen = "2.0"
-prost = { git = "https://github.com/danburkert/prost", rev = "1944c27c3029d01ff216e7b126d846b6cf8c7d77" }
-prost-build = { git = "https://github.com/danburkert/prost", rev = "1944c27c3029d01ff216e7b126d846b6cf8c7d77" }
-prost-types = { git = "https://github.com/danburkert/prost", rev = "1944c27c3029d01ff216e7b126d846b6cf8c7d77" }
+protobuf = "2"
+prost = { git = "https://github.com/danburkert/prost", rev = "1944c27c3029d01ff216e7b126d846b6cf8c7d77", optional = true }
```

Comment:
Can 0.5 be used?

Reply:
0.5 is missing some optimisations and a recursion limit. I think we need to have a 0.6 release before we can move to Prost properly (or even do useful benchmarking), but we could stick to 0.5 if you think it is important.